examples/rabbitmq_management_ssl.pp

```diff
@@ -0,0 +1,34 @@
+
+node 'rabbit.example.com' {
```

Comment:
I think either the node has to be taken out, or the rest of the code has to be indented to match.